### DISCUSSION TITLE
how to add model viewer for .glb file?

### DISCUSSION THREAD

**Discussion by Hankyu-Kim** [MicroWebStacks/astro_nav_menus#4] [Support]
I didn't know what to install, so I installed like this before I start :
`npm install three @react-three/fiber @react-three/drei`
`npm install @google/model-viewer`

and then I tried to use it exactly same way that on the demo website.
`[Astronaut](./home/Astronaut.glb)`

but here's is the error :
`[WARN] [router] A `getStaticPaths()` route pattern was matched, but no matching static path was found for requested path `/posts/sync-vs-async/@/assets/navya.glb`.    

Possible dynamic routes being matched: src/pages/posts/[...page].astro.`

what did I miss?

**Comment by wassfila** [comment]
Hi thank you for reporting an issue, from first impression, there are not enough details for reproduction and it seems that this does not relate to this repo but rather reusing it in a different project, if that is the case, then I would shift this rather to a support discussion, I'll be happy to help https://github.com/MicroWebStacks/astro-big-doc/discussions/categories/support
If this really refer to a bug using this repo as is (without code modification), then have a look at this https://github.com/MicroWebStacks/astro-big-doc?tab=readme-ov-file#user-guide if that did not work, then please provide more details and steps how to reproduce.
Note that it is not possible to fetch something that is not previously passed to getStaticPaths().
